src/components/modules/Profile/Profile.tsx

```diff
@@ -42,7 +68,20 @@ const NEW_USER_QUERY = gql`
   }
 `
 
-const Profile: React.FC<PropType> = () => {
+const Profile: React.FC<PropType> = ({ me }) => {
+  const [isEditProfile, setEditProfile] = useState(false)
+  const classes = useStyles()
+  const currentUserId = useMemo(() => _get(me, 'uid'), [me])
+  const { loading: queryLoading, error: queryError, data: queryData } = useQuery(USER_QUERY, {
+    variables: { id: currentUserId },
+  })
+
+  const meData: TApi_Users = _get(queryData, 'users.[0]')
+
+  const userFullName = `${_upperFirst(_get(meData, 'last_name'))}  ${_upperFirst(_get(meData, 'first_name'))}`
+  const email = _get(meData, 'email')
+  const country = _upperFirst(_get(meData, 'country'))
```

Comment:
Why don't you:
`const {country, email, last_name, first_name} = meData?.users?.[0] || {}`
?

Reply:
It is preferable to use get, get is safer declarative and for your example you could do it with get as follow : 
```js
_gat(meData, 'users.[0]', {})
```